packages/issm/package.py

```diff
@@ -105,16 +106,10 @@ def configure_args(self):
             args.append("--with-python-numpy-dir={0}".format(numpy_include_dir))
         else:
             args.append("--with-wrappers=no")
-
         
         return args
     
     def install(self, spec, prefix):
         # Run the normal Autotools install logic
         make("install", parallel=False)
 
-        # Copy only the examples directory directly into the prefix directory
-        examples_src = join_path(self.stage.source_path, 'examples')
-        examples_dst = join_path(prefix, 'examples')
-        install_tree(examples_src, examples_dst)
-
```

Comment:
If you want to retain this capability for installs other than the deployments we're doing centrally I'd be fine with making this a variant, e.g. `+examples` and have it default to off.